### PR TITLE
authentication and home page conversion to react

### PR DESCRIPTION
**Pull request by Naveenchandar**
login, signup, home ui and api integration changes and conversion from static to react

### CODE REVIEW

File: src/components/navbar/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                <nav className="nav_bar flex_row justify_spacebtw">
+                    <div className="nav_options">
+                        <Link to="/" className="navbar_brand nav_link align_center">
+                            Retro Cart
+                        </Link>
+                    </div>
+                    {/* <div className="nav_search flex align_center justify_center">
+                        <input type="search" placeholder="Search Products" className="input" aria-label="Search Products" />
+                    </div> */}
+                    <div className="nav_actions flex_row">
```
Comment (surbhi84) [inline src/components/navbar/index.js]: you can remove commented code if not needed

File: src/components/auth/signup.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                            {/* <div className="login_checkbox flex justify_spacebtw my-1">
+                                <div>
+                                    <input type="checkbox" />&ensp;
+                                    <span>I accept all Terms & conditions</span>
+                                </div>
+                            </div> */}
```
Comment (surbhi84) [inline src/components/auth/signup.js]: commented code

File: src/components/auth/login.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                            {/* <div className="login_checkbox flex justify_spacebtw my-1">
+                                <div>
+                                    <input type="checkbox" /> &ensp;
+                                    <span>Remember me</span>
+                                </div>
+                                <div>
+                                    <label className="primary_text_color">Forgot your password ?</label>
+                                </div>
+                            </div> */}
+                            <button
```
Comment (surbhi84) [inline src/components/auth/login.js]: .

**Review by surbhi84 — APPROVED** [review]
Navbar and footer component can be placed in your navroutes, that way you dont have to render it explicitly in each page or component
you can remove logo.png.
The folder structure looks good.

**Review by maitrakhatri — APPROVED** [review]
All the logic functions code is very easy to read.
Just a suggestion: commit the initial setup and assets in different branch as they don't need review. Also you can remove the unnecessary files like react and mockbee logo

**Pull request merged**